iife.js

```diff
@@ -0,0 +1,33 @@
+var Carlot = (function (onScreen) {
+
+onScreen.populatePage = function ()  {
+  var inventoryBox = document.querySelector('#inventoryBox .row')
+  var inventToLoad = onScreen.getJson()
+
+  console.log(">>>>", inventToLoad)
+  inventToLoad.forEach(function (car) {
+    inventoryBox.innerHTML += 
+    `<div class="col-md-3 success">
```

Comment:
Check your HTML. Each row should have three cards, and this should be done dynamically. This means that if you were to add a couple of cars to your inventory they should be in a new row.